src/lib/LedgerApi.ts

```diff
@@ -385,6 +385,7 @@ class LedgerApi {
     }
 
     // private fields and methods
+    private static readonly WALLET_ID_SALT = 42;
```

Comment:
A fixed salt still allows a pre-computed rainbow attack.

Reply:
Rainbow tables are not really feasible here. The hash function is still fed 32 bytes of random data.
But no one would use a rainbow table here anyways. What's more of a problem is, that the wallet ID is easy to calculate if the public key is known.
What are you suggesting? To create a random salt and store that in a database in the hub along with the hashes? Do you _really_ think that's worth the effort?

Reply:
No, I think hashing twice is better.

Reply:
Double hashing does not prevent the walletId being computable once the public key is known. A random salt stored in the database does. So if we decide that it should not be computable, then a random salt should be used in my opinion.